### ISSUE TITLE
Errors with the W3C validator : <button> inside <a> tags

### ISSUE THREAD

**Issue by mkrtchian** [LeonStaufer/material-dokuwiki#49]
When I try the code of a wiki page in the [W3C HTML validator](https://validator.w3.org/), it shows some errors about buttons in `<a>` tags :
> The element button must not appear as a descendant of the a element.

For buttons like [here](https://github.com/LeonStaufer/material-dokuwiki/blob/7a0572b3b12f971d352a5ab582b096e8810e5ee9/main.php#L61).

We can probably remove the `<button>` tags to let only the wrapping `<a>` tags but I guess they add some styling from material CSS, so some changes should be also done about that to keep the current styling. I haven't tried to change this yet.

**Comment by LeonStaufer** [comment]
Yes, this is an issue, mainly of semantic nature though, because it doesn't cause a problem in modern browsers.

I agree, that it would be better to only use `<a>` tags, but this would require porting a lot of styles from [MDL](https://getmdl.io/). To be honest, I'm not sure if that's worth it.

A better solution might be using something such as [`upgradeDom`](https://github.com/google/material-design-lite/issues/871) to consistently update all components of the webpage to be inline with the MDL spec.
